src/compiler/binder.ts

```diff
@@ -479,7 +479,7 @@ namespace ts {
             // and block-container.  Then after we pop out of processing the children, we restore
             // these saved values.
             const saveContainer = container;
-            const saveContainerContainer = thisParentContainer;
+            const saveThisContainer = thisParentContainer;

```

Comment:
Nit: `saveThisParentContainer`, since the `thisContainer` would just be `container` usually.